Lib/test/test_bdb.py

```diff
@@ -951,6 +949,45 @@ def test_clear_at_no_bp(self):
         with TracerRun(self) as tracer:
             self.assertRaises(BdbError, tracer.runcall, tfunc_import)
 
+    def test_load_bps_from_previous_Bdb_instance(self):
```

Comment:
A few short comments and/or breaking this up into blocks could help with readability.